.github/workflows/functions-deploy.yml

```diff
@@ -75,3 +75,7 @@ jobs:
           app-name: ${{ env.AZURE_FUNCTIONAPP_NAME }}
           package: ${{ env.AZURE_FUNCTIONAPP_PACKAGE_PATH }}
           publish-profile: ${{ steps.getPublishProfile.outputs.PUBLISH_PROFILE }}
+
+      - name: 'Validate that Azure FunctionApp loaded Functions'
```

Comment:
Depending on how the output looks, we could add some error handling or output - but I think maybe you all went down this path already? 

Either way, here is some food for thought. This probably doesn't run but a rough idea:    

functions=$(az functionapp function list -g "csels-rsti-${{ inputs.TF_ENVIRONMENT }}-moderate-rg" -n "polling-function-${{ inputs.TF_ENVIRONMENT }}" --output json)
    if [ -z "$functions" ]; then
      echo "Error: No functions found or failed to retrieve functions from Azure FunctionApp."
      exit 1
    else
      echo "Azure FunctionApp functions loaded successfully."
      echo "$functions"
    fi

Reply:
Jeff and Sam were originally implementing something similar but discovered when the command runs, if the function app function isn't found, it'll error and exit immediately.